{"uuid": "908e1eec-d0b3-4090-b450-7f3db9e572b5", "children": ["8633c5ba-bb90-414f-97bb-692144e6e797"], "befores": [{"name": "base_fixture", "status": "broken", "statusDetails": {"message": "NameError: name 'allure' is not defined\n", "trace": "  File \"C:\\Users\\SESA243476\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\SESA243476\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\SESA243476\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\SESA243476\\Documents\\JavaStudyPlan\\EclipseProject\\POM\\conftest.py\", line 16, in base_fixture\n    with allure.step(\"Initializing block....\"):\n"}, "start": 1630831762758, "stop": 1630831762758}], "start": 1630831762758, "stop": 1630831762895}